app/api/db/db.py

```diff
@@ -19,7 +20,9 @@ def __init__(self):
         self.user_contacts = self.dynamodb.Table("bs-user-contacts")
 
     @classmethod
-    def save_prediction(cls, address: str, prediction: str, team: str):
+    def save_prediction(
+        cls, address: str, prediction: str, team: str
+    ) -> Union[dict[str, str], str]:
```

Comment:
This one seems strange to me. I guess it was suggested by deduction, which could probably mean that we need to review this implementation.